### PR TITLE
profiles: add test profile

### PR DESCRIPTION
**Pull request by Br1ght0ne**

### CODE REVIEW

File: tests/hpos-admin-api/default.nix [hunk 1]
```diff
@@ -1,47 +1,32 @@
...
-    documentation.enable = false;
-
-    environment.systemPackages = [
-      hpos-admin-client
-      hpos-config
-      jq
-    ];
+    environment.systemPackages = [ hpos-admin-client ];
 
     services.hpos-admin-api.enable = true;
```
Comment (peeech) [inline tests/hpos-admin-api/default.nix:11]: I am thinking that this line needs to go to test profile as well. The purpose of the test profile is to be able to start VM that is exactly the same as test environment. So all the `service.<>.enable = true;` and `services.nginx.virtualHosts.localhost.locations.<>.proxyPass` should make it to the test profile as well.

**Review by peeech — CHANGES_REQUESTED** [review]

**Review by zo-el — COMMENTED** [review]
Looks good, I agree with PJ's suggestions. Also, we were going to merge the two tests into one test right? will that be done in a different PR?

**Comment by Br1ght0ne** [comment]
@zo-el

> we were going to merge the two tests into one test

If you mean removing `holochain` test, it's part of #678.

**Comment by zo-el** [comment]
> If you mean removing `holochain` test, it's part of #678.

I mean we were going to merge the `hpos-admin-api` and `hpos-holochain-api` tests into one test file right? so that hydra runs the tests faster?

File: tests/hpos-admin-api/default.nix (hunk 1 above)
Comment (Br1ght0ne) [reply tests/hpos-admin-api/default.nix]: Should be resolved in 47a47f4.

**Review by peeech — APPROVED** [review]
Great work!

**Pull request merged**